src/frontend/settings.ts

```diff
@@ -1 +1,2 @@
 export const API_ENDPOINT = '/api';
+export const XAPI_ENDPOINT = '/xapi/';
```

Comment:
This is a stupid comment but it would be more neat to have `'/api'` and `'/xapi'` rather than mismatched terminating slashes